analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/TestHelper.cs

```diff
@@ -199,5 +204,18 @@ public static AnalyzerOptions CreateOptions(string relativePath)
             additionalText.Setup(x => x.GetText(default)).Returns(text);
             return new AnalyzerOptions(ImmutableArray.Create(additionalText.Object));
         }
+
+        public static string CreateSonarProjectConfig(string soanrProjectConfigDirectory)
+        {
+            var sonarProjectConfigPath = Path.Combine(soanrProjectConfigDirectory, "SonarProjectConfig.xml");
+            File.WriteAllText(sonarProjectConfigPath, string.Format(ProjectConfigTemplate, soanrProjectConfigDirectory));
+            return sonarProjectConfigPath;
+        }
+
+        public static void CreateFilesToAnalyze(string filesToAnalyzeDirectory, string[] filesToAnalyze)
```

Comment:
This will save you a bunch of `new [] {`
```suggestion
        public static void CreateFilesToAnalyze(string filesToAnalyzeDirectory, params string[] filesToAnalyze)
```